frontend/src/App.js

```diff
@@ -79,15 +82,20 @@ function App() {
   const [newslist, setNewslist] = useState([]);
   const [primaryTicker, setPrimaryTicker] = useState("AAPL");
   const [watchlist, setWatchlist] = useState([]);
-  const [bottomState, setBottomState] = useState(true);
+  const [, setBottomState] = useState(true);
```

Comment:
What's the purpose of `setBottomState` if the state is not being used?

Reply:
Ahh, yeah we removed this didn't we. Cleaned up in: https://github.com/ngambino0192/stocks/pull/11/commits/1e680c51e34890fa1530ef09e5fb109317ec0edd